src/main/scala/edu/ie3/simona/agent/participant/evcs/EvcsAgentFundamentals.scala

```diff
@@ -528,6 +576,12 @@ protected trait EvcsAgentFundamentals
         modelBaseStateData
     }
 
+    determineResultsAnnounceUpdateValueStore(
+      lastState,
+      currentTick,
+      modelBaseStateData,
+    )
```

Comment:
Also, because I keep having to search for this: *This* is the most important part of the PR, I think, because it fixes results not being generated if several arrivals happen in succession without any departure. Before this PR, results were only generated when EVs depart.